examples/headsail-bsp/src/lib.rs

```diff
@@ -43,11 +43,17 @@ mod sysctrl;
 #[cfg(feature = "panic-uart")]
 mod ufmt_panic;
 
+#[cfg(feature = "hpc")]
+const EXTERNAL_BIT: usize = 0x1_0000_0000;
+
 /// # Safety
 ///
 /// Unaligned reads may fail to produce expected results on RISC-V.
 #[inline(always)]
 pub unsafe fn read_u8(addr: usize) -> u8 {
```

Comment:
Oh yeah this won't work :smiley: Do not change the behavior of `read_*` and `write_*` methods like this. It's not trivial that the external bit should always be applied to *all* accesses from HPC. Internal access is preferred when applicable. Whether or not to pass in the external bit should be managed by either the user of these functions or the memory map.

Reply:
This has been removed